### ISSUE TITLE
Support Auto-Download from CurseForge.

### ISSUE THREAD

**Issue by RyanHakurei** [itzg/docker-minecraft-server#2293]
### Enhancement Type

Improve an existing feature

### Describe the enhancement

According to the [documentation](https://docker-minecraft-server.readthedocs.io/en/latest/mods-and-plugins/modrinth/) it's possible to automatically download mods from Modrinth by giving the mod project name or project slug (the mod ID?). I am wondering if this feature is planned for CurseForge as well. The Prism launcher is able to download from CurseForge, and looking at one of the toml files it creates CurseForge does have mod IDs as well
```
[update]
[update.curseforge]
file-id = 4462358
project-id = 290209
```

So it should be possible to implement this for CurseForge too since according to #1946 this Docker image can already download modpacks from CurseForge.

**Comment by itzg** [comment]
To clarify, also from docs, yes CurseForge modpack support is indeed very much possible 

https://docker-minecraft-server.readthedocs.io/types-and-platforms/mod-platforms/auto-curseforge/

There might already be an issue to track this, but I might be confusing myself with the intention to support per mod and not just modpack support 😀

**Comment by itzg** [comment]
Resolved by #2322 https://docker-minecraft-server.readthedocs.io/en/latest/mods-and-plugins/curseforge-files/
